website/docs/reference/node-selection/test-selection-examples.md

```diff
@@ -60,32 +62,38 @@ The "buildable" and "cautious" modes can be useful in environments when you're o
 
 </VersionBlock>
 
-<VersionBlock firstVersion="1.5" >
 
-These are the modes to configure the behavior when performing indirect selection (with `eager` as the default):
+<IndirSelect features={'/snippets/indirect-selection-definitions.md'}/>
 
-1. `eager` (default) - include ANY test that references the selected nodes
-1. `cautious` - restrict to tests that ONLY refer to selected nodes
-1. `buildable` -  restrict to tests that ONLY refer to selected nodes (or their ancestors)
-1. `empty` - restrict to tests that are only for the selected node and ignore all tests from the attached nodes 
 
-Note that test exclusion is always greedy: if ANY parent is explicitly excluded, the test will be excluded as well.
+<!--tabs for eager mode, cautious mode, empty, and buildable mode -->
+<!--Tabs for 1.5+ -->
 
-The "buildable", "cautious", and "empty" modes can be useful in environments when you're only building a subset of your DAG, and you want to avoid test failures in "eager" mode caused by unbuilt resources. (Another way to achieve this is with [deferral](/reference/node-selection/defer)).
+### Indirect selection examples
 
-</VersionBlock>
+<VersionBlock firstVersion="1.5">
 
-<!--tabs for eager mode, cautious mode, and buildable mode -->
+To visualize these methods, suppose you have `model_a`, `model_b`, and `model_c` and associated data tests. The following illustrates which tests will be run when you execute `dbt run` versus `dbt build` with the various indirect selection modes:
```

Comment:
`dbt run` never selects or executes any tests, so it doesn't make sense to compare `dbt run` to `dbt build` in this context.

Rather, it only makes sense to vary:
1. Which models are selected (e.g., all vs. one).
1. Which indirect selection mode is chosen.

So we should remove mention of `dbt run` entirely:

```suggestion
To visualize these methods, suppose you have `model_a`, `model_b`, and `model_c` and associated data tests. The following illustrates which tests will be run when you execute `dbt build` with the various indirect selection modes:
```